### PR TITLE
Connect paymnet: show error when link has expired

### PR DESCRIPTION
**Pull request by robinandeer**
## Describe your changes

- Show error for expired payment links

## Justify why they are needed

Improved UX

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by robinandeer** [comment]
Current dependencies on/for this PR:
* main
  * **PR #2524** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/2524" target="_blank">Graphite</a>  👈
    * **PR #2525** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/2525" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/2524?utm_source=stack-comment).

File: apps/store/public/locales/en/common.json [hunk 1]
```diff
@@ -31,6 +31,7 @@
   "NAVIGATION_STORE_LINK": "Browse all insurances",
   "NAV_MENU_DIALOG_CLOSE": "Close",
   "NAV_MENU_DIALOG_OPEN": "Menu",
+  "PAYMENT_CONNECT_ERROR_LINK_EXPIRED": "The link is has already expired",
```
Comment (gustaveen) [inline apps/store/public/locales/en/common.json:34]: nit: Wrong in Lokalise 
```suggestion
  "PAYMENT_CONNECT_ERROR_LINK_EXPIRED": "The link has already expired",
```

**Review by gustaveen — APPROVED** [review]
Great! 🙌

**Comment by robinandeer** [comment]
@robinandeer started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/2524).

**Comment by robinandeer** [comment]
@robinandeer merged this pull request with [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/2524).

**Pull request merged**